paymentsheet/src/test/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerTest.kt

```diff
@@ -1056,6 +1058,132 @@ internal class DefaultFlowControllerTest {
         )
     }
 
+    @Test
+    fun `Returns failure if attempting to confirm while configure calls is in-flight`() = runTest {
+        val mockLoader = RelayingPaymentSheetLoader()
+        val flowController = createFlowController(paymentSheetLoader = mockLoader)
+
+        mockLoader.enqueueSuccess()
+
+        flowController.configureWithPaymentIntent(
+            paymentIntentClientSecret = PaymentSheetFixtures.CLIENT_SECRET
+        ) { _, _ -> }
```

Comment:
Can we do something in these callbacks? I'm guessing we don't expect them to be called. Maybe `error`?